### ISSUE TITLE
JVM target error

### ISSUE THREAD

**Issue by Tavorcl** [SuddenH4X/awesome-app-rating#53]
Hi, when i add the library and i try to compile my project, i get many of output lines (from gradle) that throws this error:

`Cannot inline bytecode built with JVM target 1.8 into bytecode that is being built with JVM target 1.6. Please specify proper '-jvm-target' option`

Perhaps the library's gradle file is missing this piece of code?

```
compileOptions {
    sourceCompatibility JavaVersion.VERSION_1_8
    targetCompatibility JavaVersion.VERSION_1_8
  }
```
Obviously in my project this configuration is present, but as far as i can see, the library hasn't it. And the issue only happens if i add the library in the project's gradle file, otherwise i don't get this error.

I tried some suggestions mentioned [here](https://stackoverflow.com/questions/48988778/cannot-inline-bytecode-built-with-jvm-target-1-8-into-bytecode-that-is-being-bui) but nothing seems to resolve it.

**Comment by SuddenH4X** [comment]
Hi @Tavorcl, thanks for opening this issue. You're right. I will fix this issue with the next release 😉

**Comment by SuddenH4X** [comment]
Fixed with #58.
